cert-manager/cert-manager-kube-system-resources/overlays/application/kustomization.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kube-system
+bases:
+- ../../base
+commonLabels:
+  kustomize.component: cert-manager
+configMapGenerator:
+- name: cert-manager-webhook-role-binding-parameters
+  behavior: merge
+  env: params.env
+generatorOptions:
+  disableNameSuffixHash: true
+vars:
+- name: generateName
+  objref:
+    kind: ConfigMap
+    name: cert-manager-webhook-role-binding-parameters
+    apiVersion: v1
+  fieldref:
+    fieldpath: data.generateName
+commonLabels:
+  app.kubernetes.io/name: cert-manager-webhook
+  app.kubernetes.io/instance: $(generateName)
```

Comment:
this has been changed to remove $(generateName).

So it should look like
```
  app.kubernetes.io/name: cert-manager-webhook
  app.kubernetes.io/instance: cert-manager-webhook-v0.7.0
  app.kubernetes.io/managed-by: kfctl
  app.kubernetes.io/component: cert-manager
  app.kubernetes.io/part-of: kubeflow
  app.kubernetes.io/version: v0.7.0
```

Reply:
Sure, I will change this part.